src/execution/index/art/art.cpp

```diff
@@ -898,7 +898,7 @@ BlockPointer ART::Serialize(duckdb::MetaBlockWriter &writer) {
 	if (tree) {
 		serialized_data_pointer = tree->Serialize(*this, writer);
 	} else {
-		serialized_data_pointer = {(block_id_t)DConstants::INVALID_INDEX, (uint32_t)DConstants::INVALID_INDEX};
+		serialized_data_pointer = BlockPointer::Invalid();
```

Comment:
I put this in because I need it for a future PR for the RowIDLeaf, it only really changes the "invalid" representation of the BlockPointer to not also set the `offset` field to INVALID_INDEX.

Not sure if this impacts the storage version